lib/new_relic/agent/instrumentation/fiber/chain.rb

```diff
@@ -10,9 +10,16 @@ def self.instrument!
 
         alias_method(:initialize_without_new_relic, :initialize)
 
-        def initialize(*args, &block)
-          traced_block = add_thread_tracing(*args, &block)
-          initialize_with_newrelic_tracing { initialize_without_new_relic(*args, &traced_block) }
+        if RUBY_VERSION < '2.7.0'
+          def initialize(*args, &block)
```

Comment:
I think `*args` should be `*_args` here to convey that we don't make use of it.

Reply:
oh right! you mentioned that earlier and I totally blanked on it, my bad. thanks for the reminder.
updated in https://github.com/newrelic/newrelic-ruby-agent/pull/2083/commits/ec8a03b12a23daeebf96a737ecd9cdc71a6fb922